modules/tinymce/src/plugins/table/main/ts/selection/TableSelection.ts

```diff
@@ -24,14 +34,12 @@ const getSelectionFromSelector = <T extends Element>(selector: string) =>
     return SelectorFind.closest<T>(cell, selector, isRoot);
   };
 
-const getSelectionCaption = getSelectionFromSelector<HTMLTableCaptionElement>('caption');
-
 const getSelectionCellOrCaption = getSelectionFromSelector<HTMLTableCellElement | HTMLTableCaptionElement>('th,td,caption');
 
 const getSelectionCell = getSelectionFromSelector<HTMLTableCellElement>('th,td');
 
-const getCellsFromSelection = (selections: Selections): SugarElement<HTMLTableCellElement>[] =>
-  CellOpSelection.selection(selections);
+const getCellsFromSelection = (editor: Editor): SugarElement<HTMLTableCellElement>[] =>
+  Arr.map(editor.model.table.getSelectedCells(), SugarElement.fromDom);
```

Comment:
I know this is a duplicate, but just to be sure it's not missed we probably should be using `SugarElements.fromDom` instead.